nncf/config/schemata/algo/filter_pruning.py

```diff
@@ -0,0 +1,162 @@
+                                                            "pruning level. ",
+                                                enum=FILTER_PRUNING_SCHEDULE_OPTIONS,
+                                                default=PRUNING_SCHEDULE),
+
+                    "num_init_steps": with_attributes(NUMBER,
+                                                      description="Number of epochs for model pretraining before "
+                                                                  "starting filter pruning.",
+                                                      default=PRUNING_NUM_INIT_STEPS),
+
+                    "interlayer_ranking_type": with_attributes(STRING,
+                                                               description="The type of filter ranking across the "
+                                                                           "layers.",
+                                                               enum=INTERLAYER_RANKING_TYPE_OPTIONS,
+                                                               default=PRUNING_INTERLAYER_RANKING_TYPE),
+                    "all_weights": with_attributes(BOOLEAN,
+                                                   description="Whether to prune layers independently (choose filters "
+                                                               "with the smallest importance in each layer separately) "
+                                                               "or not.",
+                                                   default=PRUNING_ALL_WEIGHTS),
+                    "prune_first_conv": with_attributes(BOOLEAN,
+                                                        description="Whether to prune first convolutional layers or "
+                                                                    "not. A 'first' convolutional layer is such a "
+                                                                    "layer that the path from model input to "
+                                                                    "this layer has no other "
+                                                                    "convolution operations on it.",
+                                                        default=PRUNE_FIRST_CONV),
+                    "prune_downsample_convs": with_attributes(BOOLEAN,
+                                                              description="Whether to prune downsampling "
+                                                                          "convolutional layers (with stride > 1) "
+                                                                          "or not. ",
+                                                              default=PRUNE_DOWNSAMPLE_CONVS),
+                    "prune_batch_norms": with_attributes(BOOLEAN,
+                                                         description="Whether to nullify parameters of Batch Norm "
```

Comment:
To unify with previous notation
```suggestion
                                                         description="Whether to prune parameters of Batch Norm "
```